src/Archive.php

```diff
@@ -97,7 +98,7 @@ public function getBin($compress = null) {
             throw new Exception("Error Processing Request", 18);
         }
 
-        return getExternal($formatBin);
+        return Utility:getExternal($formatBin);
```

Comment:
web_1       | [Thu Jan 13 05:46:29.079229 2022] [php7:emerg] [pid 25] [client 192.168.8.1:59431] PHP Parse error:  syntax error, unexpected ':', expecting ';' in /usr/src/app/plugins/filemanager/src/Archive.php on line 101, referer: http://192.168.8.135/